src/script_opt/GenRDs.h

```diff
@@ -50,9 +50,12 @@ class BlockDefs {
 
 class RD_Decorate : public TraversalCallback {
 public:
-	RD_Decorate(std::shared_ptr<ProfileFunc> _pf) : pf(std::move(_pf))
-		{ }
+	RD_Decorate(std::shared_ptr<ProfileFunc> _pf, const Func* f,
+	            ScopePtr scope, StmtPtr body);
```

Comment:
Indentation

Reply:
Didn't see the problem here ... ?

Reply:
Huh, I swear that was indented wrong. I was getting near the end here and it was all blurring together 😄